heroku/cli
# refactor(pg-v5): Move command pg:reset to oclif

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001bDP2vYAG/view

This converts `pg:reset` to oclif and updates its associated tests.

### Testing

Be careful to choose a test app. This command will literally delete all the contents of a database.


1) Pull down this branch
2) Run `./bin/run pg:reset --app APP` with an app that has a postgres db you can safely delete the contents of
3) Verify the command succeeds
4) Compare with prod
